replace-string.js

```diff
@@ -3,7 +3,7 @@ let myString = 'Hello World!!!';
 // TODO: replace the first occurence of 'o' in `myString` defined above.
 // NOTE: DO NOT HARD CODE THE SIZE OF `myString`. When you change the value
 // of `myString`, the test should still pass.
-//let newString = undefined;
+// let newString = undefined;
 
 let newString = myString.replace("Hello World!!!", "Hell0 World!!!");

```

Comment:
This is not correct. You just need to replace the `o` with `0` so the right code should be `myString.replace('o', '0');`